vm/src/hint_processor/cairo_1_hint_processor/hint_processor.rs

```diff
@@ -266,6 +266,20 @@ impl Cairo1HintProcessor {
                 t_or_k0,
                 t_or_k1,
             ),
+            Hint::Starknet(StarknetHint::Cheatcode { selector, .. }) => {
+                let selector = &selector.value.to_bytes_be().1;
+                let selector = std::str::from_utf8(selector).map_err(|_| {
```

Comment:
```suggestion
                let selector = crate::stdlib::str::from_utf8(selector).map_err(|_| {
```
We need to make this no-std compatible

Reply:
Done.